nessie/delete_objects_test.go

```diff
@@ -0,0 +1,55 @@
+package nessie
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const numOfObjects = 10
```

Comment:
can you move this one into the testing scope

Reply:
Done.